app/get-page.js

```diff
@@ -42,33 +47,60 @@ async function getPageContents (url, requestMethod) {
   return {
     contents: content.toString(),
     status: {
-      'url': url,
-      'content_type': response.headers['content-type'],
-      'content_length': contentLength,
-      'http_code': response.statusCode,
-    }
+      url: url,
+      content_type: response.headers['content-type'],
+      content_length: contentLength,
+      http_code: response.statusCode,
+    },
   }
 }
 
 async function request (url, requestMethod) {
   try {
-    const options = {
-      'method': requestMethod,
-      'encoding': null,
-      'headers': {'user-agent': process.env.USER_AGENT || DEFAULT_USER_AGENT}
+    let options, response
+    const dat = await redisGet(url + requestMethod)
+    if (dat) {
+      const body = JSON.parse(dat).body
+      const e = JSON.parse(dat).etag
```

Comment:
`JSON.parse` does not need to be called twice here
```suggestion
      const {body, etag: e} = JSON.parse(dat)
```